src/seml/database.py

```diff
@@ -18,6 +20,44 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
     return db[collection_name]
 
 
+def export_collection(collection_name, path=None):
+    collection = get_collection(collection_name)
+    cursor = collection.find({})
+    if path == None:
+        path = f'{collection_name}.bson'
+    with open(path, 'w') as file:
```

Comment:
we probably have to do `os.path.expanduser(os.path.expandvars())`, right?

Reply:
done